INSTRUCTIONS.md

```diff
@@ -46,9 +46,9 @@ Edit the newly created `.env.local` to set the app [environment variables](./fro
 Optional: you can generate a set of contract addresses for the `.env.local` file by running the following command:
 
 ```bash
-cd contracts
+cd ../../contracts
```

Comment:
I made all the `cd` instructions relative to the root directory, we tried using relative paths (https://github.com/liquity/bold/pull/738) but some steps are optional which makes it difficult to follow. Maybe we could use something like `ROOT/contracts` for all paths to make it super explicit?